src/Opdex.Platform.Infrastructure/Clients/SignalR/PlatformHub.cs

```diff
@@ -26,9 +26,10 @@ public PlatformHub(ITwoWayEncryptionProvider twoWayEncryptionProvider, AuthConfi
     /// <returns>Stratis ID.</returns>
     public string GetStratisId()
     {
-        var encryptedConnectionId = _twoWayEncryptionProvider.Encrypt(Context.ConnectionId);
+        var expiry = DateTimeOffset.UtcNow.AddMinutes(5).ToUnixTimeSeconds();
+        var encryptedConnectionId = _twoWayEncryptionProvider.Encrypt($"{Context.ConnectionId}--{expiry}");
```

Comment:
Very minor - we can potentially save 2 characters by encrypting `$"{expiry}{Context.ConnectionId}"` since we know unix time is always going to be 10 chars